collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_info.py

```diff
@@ -804,6 +804,7 @@ def generate_bucket_dict(blade):
 
 def generate_fs_dict(blade):
     fs_info = {}
+    fpolicy = blade.file_systems.list_filesystem_policies()
```

Comment:
This needs to be in a section to check the API version is high enough to support replication and policies.